app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
I think we should somewhere test that the first name of a person shows up if they are verified. View spec seems like a good place to do that.

As a related thought, why did we extract this into a method? It is only used once so feels a bit like unnecessary indirection. But I may be missing something...